tests/docker/run.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/bash
 
-rm -rf /app/target/ || true
+set -e
+
+CLEAN_BUILD=true
+
+if [ $1 = "no-clean" ]; then
+    echo "INFO: clean build is NOT going to be performed."
+    CLEAN_BUILD=false
+    find /app/target/debug/deps -name *.gcda  -exec rm {} \;
+fi
+
+if $CLEAN_BUILD ; then
+    rm -rf /app/target/ || true
+fi
```

Comment:
This helps if you want to run the tests multiple times, without waiting for a clean build. You'd just pass `no-clean` as an argument.